### ISSUE TITLE
SimCLR Finetuning Validation Loss and Finetuned Accuracy is Worse than Random

### ISSUE THREAD

**Issue by RudyVenguswamy** [PyTorchLightning/pytorch-lightning-bolts#358]
**Context:** We’re a team trying to build a reverse image searcher for satellite data. Currently, we’re using the UC Merced land use dataset (linked in notebook below) to train a self-supervised learner and evaluate it using the labels provided with the dataset. 

**Issue:** With our model that’s been self-supervised trained already, we try to finetune it using 1024 hidden dimensions (we have also tried 0, 100 hidden dims). The validation loss increases while our training loss goes to zero. There doesn’t ever seem to be an epoch at which validation loss goes down substantially and makes us suspect this isn’t a case of simple overfitting. The model saves with the lightning logs to a size of ~1gb. We have also tried fewer hidden dimensions to no avail. The validation accuracy falls to below random (acc: ~0.02).

**Code to Reproduce (Colab):** https://colab.research.google.com/drive/1C7_qkUOUHuxzq6aBkyZQm1LecQ6TJNSx?usp=sharing

**Comment by ananyahjha93** [comment]
@RudyVenguswamy from the colab notebook I can see that you are pre-training the simclr model for 15 epochs on "UC Merced land use dataset" which contains 100 images per class for 21 classes. 

I think this insufficient pre-training for a model like SimCLR both in terms of data and training time. The actual SimCLR pre-training is done with ~1.2 million images from ImageNet for around 800 epochs.

Also, using pre-trained SimCLR weights to fine-tune on "UC Merced land use dataset" might be something that you can try but I think there is a significant shift between the distributions of images from ImageNet and satellite images present in this dataset.

**Comment by ananyahjha93** [comment]
Closing this issue for now! Can merge in #356
